### PR TITLE
feat: empty portfolio allow to receive without an asset selected

### PR DESCRIPTION
**Pull request by reallybeard**
## Description

- Needed a way to receive an asset without one selected
- Added empty account list
- Fixed some key warnings

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [ ] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?
- [ ] Do all new and existing tests pass? Does the linter pass?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

If applicable, please link to the github issue and put `closes #XXXX` in your comment to auto-close the issue that your PR fixes.

## Testing

Please outline all testing steps

1. Pull branch locally and run `yarn` to install new deps
2. etc...

## Screenshots (if applicable
Screen Shot 2021-11-18 at 4 35 57 PM
Screen Shot 2021-11-18 at 4 36 02 PM
Screen Shot 2021-11-18 at 4 36 06 PM
)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,7 +68,8 @@
     "redux-persist": "^6.0.0",
     "scrypt-js": "^3.0.1",
     "web-vitals": "^2.1.0",
-    "web3": "^1.6.0"
+    "web3": "^1.6.0",
+    "yarn": "^1.22.17"
```
Comment (0xdef1cafe) [inline package.json:72]: ?

File: package.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
     "@visx/responsive": "^1.10.1",
     "@visx/shape": "^1.17.1",
     "@visx/tooltip": "^1.17.1",
+    "add": "^2.0.6",
```
Comment (0xdef1cafe) [inline package.json:33]: ?

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
yarn add yarn add?

**Review by elmutt — APPROVED** [review]

**Comment by 0xdef1cafe** [comment]
tested and this look fuckin rad we should merge today

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**